lib/internal/bootstrap_loaders.js

```diff
@@ -0,0 +1,230 @@
+// This file creates the internal module & binding loaders used by built-in
+// modules. In contrast, user land modules are loaded using
+// lib/internal/module.js (CommonJS Modules) or
```

Comment:
I think at this point `lib/internal/module.js` mostly contains helper functions for the loader in `lib/module.js`, so it might be better to point to that?

Reply:
@addaleax Yeah, `lib/module.js` seems to be better (also, we should probably rename those files to make it clear which one is loading what..)